tests/o3/cartesian_spherical_harmonics_test.py

```diff
@@ -156,6 +157,38 @@ def build_module(l, normalize, normalization):
     xyz = torch.randn(11, 3)
     assert torch.allclose(sp_jit(xyz), o3.spherical_harmonics(l, xyz, normalize, normalization))
     assert_equivariant(sp)
-
+    torch._dynamo.reset()
     sp_pt2 = torch.compile(prepare(build_module)(l, normalize, normalization), fullgraph=True)
     assert torch.allclose(sp_pt2(xyz), sp(xyz))
+
+
+def test_internal_jit_flag():
+    l = o3.Irreps("0e + 1o + 3o")
+    sp = o3.SphericalHarmonics(l, normalization="integral", normalize=True)
+    # Turning off the JIT in _spherical_harmonics to enable torch.compile.
+    # Note that this is a less invasive way then turning off jit_script_fx globally
+    try:
+        torch._dynamo.reset()
```

Comment:
Why is this necessary?

Reply:
The compiler cache gets built up from previous runs so need to flesh that out

Reply:
Looks like bumping to 2.2.0 -> 2.4.0 fixed that one